docs/js/alert.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Alert
+layout: page.jade
+sidebar: true
+collection: js
+priority: 3
+path: alert
+---
+
+# Alert
+<p class="lead">The Alert component is an empty container that you input a message and have an OK button by default.</p>
```

Comment:
Tá bem melhor, mas vê se assim fica melhor: `The alert component is an empty container where you can add a message that will be presented to users. It has an OK button by default.`